packages/financial-templates-lib/test/truffle/BalancerPriceFeed.js

```diff
@@ -0,0 +1,430 @@
+const { toWei, toBN } = web3.utils;
```

Comment:
I actually really like that this is a copy-paste of the uniswap tests, since it basically ensures that the behavior matches exactly.

I could imagine that there are some edge cases that are unique to each implementation that we want to test, but overall, I think it really makes sense to hit them with the same test cases.

Reply:
I'd like to eventually merge these two tests into a `TwapPriceFeed` generalized test that should work for any twap'd price feed.